pkgs/applications/misc/gpxsee/default.nix

```diff
@@ -18,13 +18,13 @@ let
 in
 stdenv.mkDerivation (finalAttrs: {
   pname = "gpxsee";
-  version = "13.40";
+  version = "13.42";
```

Comment:
Is this change on purpose? What's the reasoning? Is it related to davis?

Reply:
*sigh* this happens when you do a `git rebase -i`  to squash or reword commits, and then forget to do a `git rebase upstream/master` afterwards. Fixed!